frame/assets/src/functions.rs

```diff
@@ -812,6 +861,192 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 		Ok(())
 	}
 
+	/// Disallow further unprivileged transfers for the asset class.
+	///
+	/// Origin must be Signed and the sender should be the Freezer of the asset `id`.
+	///
+	/// - `id`: The identifier of the asset to be frozen.
+	///
```

Comment:
If you are going to comment the parameters, you should at least include all of them.